runtime/darwinia-parachain/src/pallets/helixbridge.rs

```diff
@@ -0,0 +1,62 @@
+// --- crates.io ---
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+// --- paritytech --
+use frame_support::{PalletId, RuntimeDebug};
+// --- darwinia-network ---
+use crate::*;
+use bp_message_dispatch::CallOrigin;
+use bp_messages::{LaneId, MessageNonce};
+use bp_runtime::{messages::DispatchFeePayment, ChainId, DARWINIA_CHAIN_ID};
+use bridge_runtime_common::lanes::DARWINIA_DARWINIA_PARACHAIN_LANE;
+use bridges_message::darwinia::ToDarwiniaMessagePayload;
+use dp_common_runtime::helixbridge::{CallParams, Config, CreatePayload, LatestMessageNoncer};
+use pallet_bridge_messages::{outbound_lane, Instance1 as WithDarwiniaMessages};
+
+/// The s2s backing pallet index in the darwinia chain runtime.
+const DARWINIA_S2S_BACKING_PALLET_INDEX: u8 = 57;
```

Comment:
@xiaoch05 Please help review this. `57` refers to the fee-market in the latest darwinia runtime. So I think this is incorrect here.

https://github.com/darwinia-network/darwinia/blob/main/runtime/darwinia/src/lib.rs#L259

Reply:
yes, I will check all the configure variable for this pallet.